.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
When husky is installed, it adds some files to the repo that should be uploaded, like this one or the above one.